gnd/src/main/java/com/google/android/gnd/system/CameraManager.java

```diff
@@ -71,11 +70,12 @@ private Completable sendCapturePhotoIntent() {
   }
 
   /** Emits the result of the photo capture request. */
-  @Hot
-  public Observable<Bitmap> capturePhotoResult() {
+  @Hot(terminates = true)
+  public Maybe<Bitmap> capturePhotoResult() {
```

Comment:
Sorry if I created some confusion with my last comment. I think the API of this class could be simplified. Rather than sending the photo capture request and capturing it via separate methods, would it be possible to expose a single `@Cold Maybe<Bitmap> capturePhoto()` that launches the intent and waits for the result?

Reply:
Thanks for the suggestion. Changes applied.